### PR TITLE
(fix) Style fixes for search.

### PR DESCRIPTION
**Pull request by mdhaman**
- SD-2010 : Styles issues in grid view.
- SD-2009 : Layout issues in search (tags were hidden behind search results.)

### CODE REVIEW

File: client/app/scripts/superdesk-search/views/item-search.html [hunk 1]
```diff
@@ -2,13 +2,13 @@
     <fieldset ng-if="context">
         <div class="field">
             <label for="ingest-collection">
-                {{: 'Ingest' | translate}}
+                {{ 'Ingest' | translate}}
```
Comment (eozan) [inline client/app/scripts/superdesk-search/views/item-search.html]: I think it was meant to be
```{{:: 'Ingest' | translate }}```

Comment (amagdas) [inline client/app/scripts/superdesk-search/views/item-search.html]: I already merged the fix for this error.

Comment (petrjasek) [inline client/app/scripts/superdesk-search/views/item-search.html]: so can you rebase this @mdhaman ?

**Comment by mdhaman** [comment]
@petrjasek rebase done.

**Comment by petrjasek** [comment]
:+1:

**Pull request merged**